Review section about Accordion control. (#590)
telerik/maui-docs: controls/accordion/accordion-item.md

---
title: AccordionItem
page_title: .NET MAUI Accordion Documentation - AccordionItem

> description: Check our AccordionItem documentation article for Telerik .NET MAUI Accordion control.

Reviewer: The description of the article is the ad you are using to advertise it. There is nothing interesting about "Check our AccordionItem documentation article..." - this is the same as having nothing at all.

A meaningful description must answer the following questions:
* What is the described thing?
* What is the main objective of the article? What does the article tell? (Usually to learn something about a component, feature, or property)
* What are the benefits of using the described component or feature?
* What are the use cases for this component or feature? What can you do with it?

```suggestion
description: Discover the available configuration options of the Accordion Item and learn how to use them to customize the appearance and the behavior of the Telerik UI for .NET MAUI Accordion.
```

## AccordionItem Header

> You can either apply `HeaderText` property or use the `AccordionItemHeader` content control which provides a set of useful properties for customizing the look & feel of the Header.

Reviewer: ```suggestion
You can either apply the `HeaderText` property or use the `AccordionItemHeader` content control which provides a set of useful properties for customizing the look & feel of the Header.
```

> The indicator is the small triangle that is rotated according to whether the `AccordionItem` control is expanded or collapsed. `AccordionItemHeader` provides various options for customizing the look of the indicator via the following properties:

Reviewer: ```suggestion
The indicator is the small triangle that is rotated depending on whether the `AccordionItem` control is expanded or collapsed. `AccordionItemHeader` provides various options for customizing the look of the indicator via the following properties:
```

## IsExpanded

> Change the expanded state of the `AccordionItem` using the `IsExpanded`(`bool`)property.

Reviewer: When not part of the code, always use spaces between the words that precede or follow content in parentheses. This is a very common mistake in the MAUI docs.

	
## Border Styles

> Use `BorderColor`, `BorderThickness` and `CornerRadius` properties of `RadAccordionItem` to change the way the Border around the control looks.

Reviewer: Always use the definite article when referring to a specific noun or unique object!
When you describe the "property" as "BorderColor", this makes it specific. It is not any property, it is **the BorderColor property**. 
```suggestion
Use the `BorderColor`, `BorderThickness`, and `CornerRadius` properties of `RadAccordionItem` to change the way the Border around the control looks.
```
